test/xdb_lib_SUITE.erl

```diff
@@ -1,5 +1,7 @@
 -module(xdb_lib_SUITE).
 
+-dialyzer({nowarn_function, t_raise/1}).
```

Comment:
I couldn't find any other way to stop dialyzer complaining about funs being `no_return()`